engine/engine_test.go

```diff
@@ -1763,6 +1763,575 @@ func TestBinopEdgeCases(t *testing.T) {
 	testutil.Equals(t, oldResult, newResult)
 }
 
+func TestXFunctions(t *testing.T) {
+	defaultQueryTime := time.Unix(50, 0)
+	fmt.Println(defaultQueryTime.String())
```

Comment:
Shall we remove this or convert to `t.Log`?

Reply:
Good point - updated it here: 7425161cf6b2b641ac1fd107e376bed9177a9373